### PR TITLE
DCA-964:Update naming for jira actions

### PR DESCRIPTION
**Pull request by mmizin**
https://bulldog.internal.atlassian.com/browse/DCA-964

### CODE REVIEW

File: app/jmeter/jira.jmx [hunk 1]
```diff
@@ -1124,14 +1124,14 @@ JMeterUtils.setProperty(&quot;c_AtlToken&quot; + user_counter, atl_token)
             </JSR223PreProcessor>
             <hashTree/>
           </hashTree>
-          <ThroughputController guiclass="ThroughputControllerGui" testclass="ThroughputController" testname="create issue" enabled="true">
+          <ThroughputController guiclass="ThroughputControllerGui" testclass="ThroughputController" testname="create_issue" enabled="true">
```
Comment (ometelytsia) [inline app/jmeter/jira.jmx:1127]: Hmm, what is the difference here?

Comment (mmizin) [reply app/jmeter/jira.jmx:1127]: underscore I think

File: app/jmeter/jira.jmx [hunk 2]
```diff
@@ -1124,14 +1124,14 @@ JMeterUtils.setProperty(&quot;c_AtlToken&quot; + user_counter, atl_token)
...
             <intProp name="ThroughputController.style">1</intProp>
             <boolProp name="ThroughputController.perThread">false</boolProp>
             <intProp name="ThroughputController.maxThroughput">1</intProp>
             <stringProp name="ThroughputController.percentThroughput">${perc_create_issue}</stringProp>
           </ThroughputController>
           <hashTree>
-            <TransactionController guiclass="TransactionControllerGui" testclass="TransactionController" testname="jmeter_open_quick_create" enabled="true">
+            <TransactionController guiclass="TransactionControllerGui" testclass="TransactionController" testname="jmeter_create_issue:open_quick_create" enabled="true">
```
Comment (ometelytsia) [inline app/jmeter/jira.jmx:1134]: Where are other jmeter_create_issue subactions?

File: app/jmeter/jira.jmx [hunk 3]
```diff
@@ -7935,7 +7935,7 @@ vars.put(&quot;random_page&quot;, String.valueOf(random_page));</stringProp>
               </hashTree>
             </hashTree>
           </hashTree>
-          <ThroughputController guiclass="ThroughputControllerGui" testclass="ThroughputController" testname="view backlog" enabled="true">
+          <ThroughputController guiclass="ThroughputControllerGui" testclass="ThroughputController" testname="standalone_extension" enabled="true">
```
Comment (ometelytsia) [inline app/jmeter/jira.jmx:7938]: What? Is this correct? ^^

File: app/jmeter/jira.jmx (hunk 2 above)
Comment (mmizin) [reply app/jmeter/jira.jmx:1134]: image
I think in that case we don't need like that "jmeter_create_issue:create_issue". How do you think?

File: app/jmeter/jira.jmx (hunk 3 above)
Comment (mmizin) [reply app/jmeter/jira.jmx:7938]: Fixed it

File: app/jmeter/jira.jmx (hunk 2 above)
Comment (ometelytsia) [reply app/jmeter/jira.jmx:1134]: see corresponding selenium action as example

Comment (mmizin) [reply app/jmeter/jira.jmx:1134]: We have the same for Selenium

Comment (ometelytsia) [reply app/jmeter/jira.jmx:1134]: Screenshot 2020-12-22 at 16 04 46

**Review by ometelytsia — APPROVED** [review]

**Review by astashys — APPROVED** [review]

**Pull request merged**